app/app.js

```diff
@@ -0,0 +1,8 @@
+var app = angular.module('MyApp', [
+    'todo.module'
+]);
+
+app.controller('MyController', ['$scope', function ($scope) {
```

Comment:
create separate file for controller `MyController `

Reply:
done